tutorials/extragalactic_gcr_photoz_catalogs.ipynb

```diff
@@ -0,0 +1,690 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# cosmoDC2 extragalactic catalog photometric redshifts\n",
+    "Owner:**Sam Schmidt [@sschmidt23](https://github.com/LSSTDESC/DC2-analysis/issues/new?body=@sschmidt23)**\n",
+    "\n",
+    "Last Verifed to Run: **2020-06-09** (by @sschmidt23)\n",
+    "\n",
+    "This notebook will show you how to access the \"add-on\" columns that provide the photometric redshift (photo-z) information for the extragalactic catalog (cosmoDC2_v1.1.4_image). \n",
+    "\n",
+    "__Learning objectives__: After going through this notebook, you should be able to:\n",
+    "  1. Load and efficiently access a DC2 extragalactic catalog (+ photo-z) via the GCR for both a template based and machine learning-based photo-z methods\n",
+    "  2. Understand how the photo-z data are stored / represented\n",
+    "  3. Look at a few examples of galaxy photo-z distributions\n",
+    "  \n",
+    "__Logistics__: This notebook is intended to be run through the Jupyter Lab NERSC interface available here: https://jupyter.nersc.gov/. To setup your NERSC environment, please follow the instructions available here: https://confluence.slac.stanford.edu/display/LSSTDESC/Using+Jupyter+at+NERSC\n",
+    "\n",
+    "__Other notes__: \n",
+    "If you restart your kernel, or if it automatically restarts for some reason, all imports and variables will become undefined so, you will have to re-run everything.\n",
+    "Several photo-z catalogs were renamed starting with GCRCatalogs version 0.18.0, so check that GCRCatalogs is at least this recent when running the notebook.\n",
+    "\n",
+    "This notebook will be very similar to the earlier tutorial on useing the object catalogs, so if you have worked through that example much of this will be very familiar.\n",
+    "\n",
+    "A rendered version of this notebook is available at:<br>\n",
+    "https://nbviewer.jupyter.org/github/LSSTDESC/DC2-analysis/blob/rendered/tutorials/extragalactic_gcr_photoz_catalogs.nbconvert.ipynb <br>\n",
+    "if you want to follow along but do not wish to actually run the notebook live at NERSC."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Temporary path to gcr-catalogs until new version is included in desc-python kernel:\n",
+    "import sys\n",
+    "sys.path.insert(0,\"/global/cfs/cdirs/lsst/groups/PZ/Packages/gcr-catalogs/lib/python3.7/site-packages/GCRCatalogs-0.18.1-py3.7.egg\")"
+   ]
+  },
```

Comment:
Remove this cell when the kernel is updated.
```suggestion
```

Reply:
removed